lib/matplotlib/axes/_base.py

```diff
@@ -1286,26 +1286,68 @@ def apply_aspect(self, position=None):
                 self.set_xbound((x0, x1))
 
     def axis(self, *v, **kwargs):
-        """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        """Set axis properties.
+
+        If no positional or kwargs are given, returns the current
+        axes limits are returned with no side effects.
+
+        If a positional argument is passed the keyword arguments are ignored.
+
+        If no positional argument is passed, the keyword arguments are used.
```

Comment:
Can these three explanations above be replaced or enhanced with 3 signatures?  It's easier to understand
```
Valid signatures:
xmin, xmax, ymin, ymax = axis()
xmin, xmax, ymin, ymax = axis(list_arg)
xmin, xmax, ymin, ymax = axis(string_arg)
xmin, xmax, ymin, ymax = axis(**kwargs)
```
than the explanation of it.  If something like this can't go in the block above, maybe examples can be included.